Makefile

```diff
@@ -96,14 +108,6 @@ install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~
 uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/crd | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
-.PHONY: deploy
-deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
I'm wondering if the deploy and underlay targets might become useful again for modularisation. When getting rid of Helm, this might be the main flow for in-cluster testing.

Reply:
sure, this can be default flow for deploy and undeploy after creating a module from controller

Reply:
So should we keep it or remove first and add again when needed?

Reply:
they will not work any way for now, lets remove and add the both target for module